src/query/sql/src/planner/optimizer/cascades/tasks/explore_expr.rs

```diff
@@ -149,17 +153,34 @@ impl ExploreExprTask {
             .memo
             .group(self.group_index)?
             .m_expr(self.m_expr_index)?;
+        let rule_set: roaring::RoaringBitmap;
+        unsafe {
+            rule_set = m_expr
+                .plan
+                .exploration_candidate_rules()
+                .bitand(&RULE_FACTORY.exploration_rules);
+        }
```

Comment:
It's better to wrap the unsafe code with a safe function call.

Reply:
ok, I'll resolve it tonight.